doc/sphinx/language/core/index.rst

```diff
@@ -30,10 +30,67 @@ will have to check their output.
 .. toctree::
    :maxdepth: 1
 
+   .. The two chapters Gallina specification language and CIC are
+      planned for removal.  Eventually, the present "Core language"
```

Comment:
The Gallina spec chapter file goes away, but the material will be moved.
Will any of CIC be dropped entirely?  Also, expect there will be more rewriting here to make all the formulas more accessible.

Reply:
I don't think any of the current material should be removed. Making it more accessible shouldn't mean making it less complete. Besides, with the structure for each section that we proposed in the CEP, it should be OK that in each section, there are some explanations that are less accessible than others:

>Topics are gathered in a single place where they are explained in depth, starting with a high level view, with lots of examples, then a more exhaustive presentation aimed at the experts, to finish with theoretical and implementation considerations that hint at how to improve the actual code, what are the limitations, and how to contribute.